pkg/util/sketch/cms_test.go

```diff
@@ -0,0 +1,52 @@
+//go:build sketch
```

Comment:
That's fine for starters but we need functional testing that always runs.

Reply:
Yes I'll split things out into separate files.